### DISCUSSION TITLE
KNX direct ip connection

### DISCUSSION THREAD

**Discussion by koos147** [arendst/Tasmota#13806] [Ideas]
Hello all,

i am trying to connect an sonoff to and existing knx system.
however i noticed that tasmota only support multicast connections.
is there any change to connect with a direct ip connection?
at the moment i use home-assistent as "bridge" however at a new location i need to connect to the Gira homeserver. 

/offtopic
multicast isn't the right solution (for discovery it is acceptable. but should never be part of the main function of aan application)

kind regards
Mark

**Comment by ascillato** [comment]
Hi, sorry. KNX tunneling is not supported and there is no plans to support it soon. Sorry.

**Comment by koos147** [comment]
Hey Ascillato,
Thanks for the reply, so i can stop searching ways to get it working.
already found out that it is in your roadmap. however its the last point so i guess it will take a long time to get to the botom of your todo.

will follow your project. for the moment you (or someone else who can write code) will ever make it happen. :)

and please keep in mind... multicast is something made for small consumer networks.... never ever try using it in large networks :)

**Reply by ascillato** [reply]
KNX IP Multicast is part of the KNX Standard. It is usable and don't have issues if you do follow all the requirements for your Wi-Fi Router to have correct multicast broadcasting. KNX IP Tunneling don't allow much devices to the same KNX Router. Some Gira Routers allows up to 5 connections.
